config.yaml

```diff
@@ -1,8 +1,59 @@
+apiVersion: v1
```

Comment:
So, is there a reason we shouldn't make this Jsonnet instead of YAML? Also, it might be nice to see if we can wrap this into a single deployment step.